cmake/modules/BuildQATzip.cmake

```diff
@@ -11,16 +11,21 @@ function(build_qatzip)
   set(configure_cmd env CC=${CMAKE_C_COMPILER} ./configure --prefix=${QATzip_INSTALL_DIR})
   # build a static library with -fPIC that we can link into crypto/compressor plugins
   list(APPEND configure_cmd --with-pic --enable-static --disable-shared)
+
+  set(CFLAGS "-Wno-error=strict-prototypes -Wno-error=unused-but-set-variable")
```

Comment:
do you have an idea for how we will be able to go remove these workarounds if/when the QATzip library gets fixed?  (I know this isn't a new problem, but introducing more things that never get revisited always stings)

Reply:
We have dozens of -Wno flags set throughout our cmake/modules and many more in other cmake files. Going over each and every warning disabled / how we treat disabled warning is out of the scope of this PR IMO.